src/js/common/schemaform/FormApp.jsx

```diff
@@ -42,12 +42,11 @@ class FormApp extends React.Component {
     }
 
     // If we start in the middle of a form, redirect to the beginning
-    const currentPath = this.props.currentLocation.pathname;
-    const firstPagePath = this.props.routes[this.props.routes.length - 1].pageList[0].path;
     // If we're in production, we'll redirect if we start in the middle of a form
     // In development, we won't redirect unless we append the URL with `?redirect` ()
     const devRedirect = __BUILDTYPE__ !== 'development' || this.props.currentLocation.search.includes('redirect');
-    if (currentPath !== firstPagePath && devRedirect) {
+    if (isInProgress(this.props.currentLocation.pathname) && devRedirect) {
```

Comment:
I just tested with and without the trailing `/` in the URL and it worked as expected in both cases. 🤔  It doesn't seem like it _should_ work with the trailing `/`. So that's odd...